### PR TITLE
docs: migrate to antd-style

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link
https://github.com/ant-design/ant-design/issues/40235

### 💡 Background and solution

去除 `@emotion/react` 和 css prop 的写法，全量用 antd-style
全局样式的写法比较怪，去 antd-style 加个工厂方法再来
https://ant-design.github.io/antd-style/api/global-styles#%E7%BB%93%E5%90%88-antd-%E7%9A%84-token-%E4%BD%BF%E7%94%A8

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      -     |
| 🇨🇳 Chinese |     -      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 5247fb2</samp>

The pull request migrates the dumi pages and theme components from `@emotion/react` to `antd-style` for styling and theming. This improves the performance, compatibility, and consistency of the project and simplifies the code. It also fixes some minor issues with indentation, naming, and props. The unused `useSiteToken` hook and the `RecommendsOld` component were removed.

### <samp>🤖 Generated by Copilot at 5247fb2</samp>

*  Replace `@emotion/react` with `antd-style` in all import statements to use the wrapper package that provides additional features for styling components ([link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-8224af725d22f9ec09fdfbfddab99d82b2438b8fb9c4897afb52fba55d43a29fL1-R9), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-6a484b82d5234dc2b8c7c367190b951e79e2d45ebe10433597c11365aee1c877L3-R10), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-12f648feaf3caa0382edb3b014fdd7dbedcad48bee322535d7cd01c5d210c9e6L17-R21), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-78d917ce86078797614d11e0c46bdd7f0f15ff00b3ebc25bca413370f07be21aL3-R5), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-334e4791bcfe5714cefed473096aba70368d2cea07143640f8396e570b5e61f7L1-R2), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-bb71d3e7437a45c3978afff7a00abefa4e0c15a401677f1a732c773f7027d36eL1-R1), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-a0e54c8709f9bb05620ed26294aecfdbd07627aa7abb084e87ccc5a580530c71L3-R8), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-9ac0f4a6f5f6e7f8ca9a08ba42c07109b6705731c89c6b1af5daf7e2e9d5850dL1-R5), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-fb37d7abd9dddd9f2a8c076bfd9eea2336873f1051d1f37dceaca099206a4aedL7-R7), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-5137c94c78aea4423103cc7e7309ec68b1d233d0dae3b03750694ea29c3206cfL2-R2), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-30afbf329ccd8dcb6bf69a0c2b4464e6ef851c61ed0d4c133113aa6afd4b8f28L1-R1), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-ef29a310ce5cad95573716c19d940bb06da30b1a8e571b8d7f6154dd8c42d92eL1-R1), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-51d36d0662438e162135ad6713e967a78491b32e953192265b2ecc71c9800e26L2-R3), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-f1df5cf9560d26c9862a4824e7d1a13f6a8bec1ec7338e9327f3e7a6a74c1585L4-R4), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-377ed68f4e882f6438c31331026bdb11548e6b62be65727a5793cda37badfa93L2-R2), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-0257ee7455cf658a30e3d6ef9a4e98d4aeb93d6ec5d9d1421e0305718eac4dbfL7-R7))
*  Remove `useSiteToken` hook from all components and functions and replace it with `useTheme` hook from `antd-style` to access the theme token object consistently and support custom token types ([link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-8224af725d22f9ec09fdfbfddab99d82b2438b8fb9c4897afb52fba55d43a29fL26-R35), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-8224af725d22f9ec09fdfbfddab99d82b2438b8fb9c4897afb52fba55d43a29fL61-R62), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-6a484b82d5234dc2b8c7c367190b951e79e2d45ebe10433597c11365aee1c877L50-R48), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-6a484b82d5234dc2b8c7c367190b951e79e2d45ebe10433597c11365aee1c877L93-R91), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-12f648feaf3caa0382edb3b014fdd7dbedcad48bee322535d7cd01c5d210c9e6L58-R59), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-78d917ce86078797614d11e0c46bdd7f0f15ff00b3ebc25bca413370f07be21aL64-R64), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-935d4f3f1f1185da224c9f1168ce1e28a667f4374033983d472a1b1923a14a7eL4-R5), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-935d4f3f1f1185da224c9f1168ce1e28a667f4374033983d472a1b1923a14a7eL53-R53), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-844617f9965e63398da7daa55eb25905d69615fd3e95c4953b8f486661b93d0aL73-R69), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-334e4791bcfe5714cefed473096aba70368d2cea07143640f8396e570b5e61f7L11-R11), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-bb71d3e7437a45c3978afff7a00abefa4e0c15a401677f1a732c773f7027d36eL7-R11), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-bb71d3e7437a45c3978afff7a00abefa4e0c15a401677f1a732c773f7027d36eL33-R34),  [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-a0e54c8709f9bb05620ed26294aecfdbd07627aa7abb084e87ccc5a580530c71L23-R22), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-9ac0f4a6f5f6e7f8ca9a08ba42c07109b6705731c89c6b1af5daf7e2e9d5850dL31-R32), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-9ac0f4a6f5f6e7f8ca9a08ba42c07109b6705731c89c6b1af5daf7e2e9d5850dL78-R75), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-fb37d7abd9dddd9f2a8c076bfd9eea2336873f1051d1f37dceaca099206a4aedL25-R29), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-fb37d7abd9dddd9f2a8c076bfd9eea2336873f1051d1f37dceaca099206a4aedL86-R89), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-fb37d7abd9dddd9f2a8c076bfd9eea2336873f1051d1f37dceaca099206a4aedL274-R284), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-5137c94c78aea4423103cc7e7309ec68b1d233d0dae3b03750694ea29c3206cfL102-R102), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-30afbf329ccd8dcb6bf69a0c2b4464e6ef851c61ed0d4c133113aa6afd4b8f28L21-R21), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-ef29a310ce5cad95573716c19d940bb06da30b1a8e571b8d7f6154dd8c42d92eL41-R41), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-ef29a310ce5cad95573716c19d940bb06da30b1a8e571b8d7f6154dd8c42d92eL80-R80), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-3bb66880942ca129db42a1a6c20126916fc7b9be5daf9723506df03fa96f0995L19-R38), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-51d36d0662438e162135ad6713e967a78491b32e953192265b2ecc71c9800e26L11-R11), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-51d36d0662438e162135ad6713e967a78491b32e953192265b2ecc71c9800e26L22-R18), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-f1df5cf9560d26c9862a4824e7d1a13f6a8bec1ec7338e9327f3e7a6a74c1585L10-R21), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-f1df5cf9560d26c9862a4824e7d1a13f6a8bec1ec7338e9327f3e7a6a74c1585L30-R27), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-f1df5cf9560d26c9862a4824e7d1a13f6a8bec1ec7338e9327f3e7a6a74c1585L58-R55), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-f1df5cf9560d26c9862a4824e7d1a13f6a8bec1ec7338e9327f3e7a6a74c1585L82-R78), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-f1df5cf9560d26c9862a4824e7d1a13f6a8bec1ec7338e9327f3e7a6a74c1585L88-R84), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-377ed68f4e882f6438c31331026bdb11548e6b62be65727a5793cda37badfa93L9), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-377ed68f4e882f6438c31331026bdb11548e6b62be65727a5793cda37badfa93L33-R32), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-377ed68f4e882f6438c31331026bdb11548e6b62be65727a5793cda37badfa93L60-R64), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-0257ee7455cf658a30e3d6ef9a4e98d4aeb93d6ec5d9d1421e0305718eac4dbfL14), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-0257ee7455cf658a30e3d6ef9a4e98d4aeb93d6ec5d9d1421e0305718eac4dbfL24-R23), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-0257ee7455cf658a30e3d6ef9a4e98d4aeb93d6ec5d9d1421e0305718eac4dbfL57-R56), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-0257ee7455cf658a30e3d6ef9a4e98d4aeb93d6ec5d9d1421e0305718eac4dbfL115-R114))
*  Wrap `useStyle` function with `createStyles` function from `antd-style` in all components to create style objects with the theme token object and the `css` function as arguments, and return an object with the `styles` and `cx` properties ([link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-8224af725d22f9ec09fdfbfddab99d82b2438b8fb9c4897afb52fba55d43a29fL51-R51),  [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-12f648feaf3caa0382edb3b014fdd7dbedcad48bee322535d7cd01c5d210c9e6L92-R91), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-78d917ce86078797614d11e0c46bdd7f0f15ff00b3ebc25bca413370f07be21aL93-R94), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-844617f9965e63398da7daa55eb25905d69615fd3e95c4953b8f486661b93d0aL81-R77), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-334e4791bcfe5714cefed473096aba70368d2cea07143640f8396e570b5e61f7L24-R26), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-bb71d3e7437a45c3978afff7a00abefa4e0c15a401677f1a732c773f7027d36eL76-R72), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-a0e54c8709f9bb05620ed26294aecfdbd07627aa7abb084e87ccc5a580530c71L80-R86), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-9ac0f4a6f5f6e7f8ca9a08ba42c07109b6705731c89c6b1af5daf7e2e9d5850dL69-R66), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-fb37d7abd9dddd9f2a8c076bfd9eea2336873f1051d1f37dceaca099206a4aedL187-R196), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-30afbf329ccd8dcb6bf69a0c2b4464e6ef851c61ed0d4c133113aa6afd4b8f28L42-R42), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-ef29a310ce5cad95573716c19d940bb06da30b1a8e571b8d7f6154dd8c42d92eL50-R50), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-51d36d0662438e162135ad6713e967a78491b32e953192265b2ecc71c9800e26L30-R29), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-f1df5cf9560d26c9862a4824e7d1a13f6a8bec1ec7338e9327f3e7a6a74c1585L48-R45), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-377ed68f4e882f6438c31331026bdb11548e6b62be65727a5793cda37badfa93L49-R48), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-0257ee7455cf658a30e3d6ef9a4e98d4aeb93d6ec5d9d1421e0305718eac4dbfL30-R29))
*  Replace `css` prop with `className` prop in all React elements and components to apply class names in a standard way, and use `classNames` function from `classnames` package to combine multiple class names from the `styles` object ([link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-8224af725d22f9ec09fdfbfddab99d82b2438b8fb9c4897afb52fba55d43a29fL149-R149), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-6a484b82d5234dc2b8c7c367190b951e79e2d45ebe10433597c11365aee1c877L56-R58), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-6a484b82d5234dc2b8c7c367190b951e79e2d45ebe10433597c11365aee1c877L72-R70), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-6a484b82d5234dc2b8c7c367190b951e79e2d45ebe10433597c11365aee1c877L100-R98), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-6a484b82d5234dc2b8c7c367190b951e79e2d45ebe10433597c11365aee1c877L107-R105), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-6a484b82d5234dc2b8c7c367190b951e79e2d45ebe10433597c11365aee1c877L113-R111), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-6a484b82d5234dc2b8c7c367190b951e79e2d45ebe10433597c11365aee1c877L119-R117), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-12f648feaf3caa0382edb3b014fdd7dbedcad48bee322535d7cd01c5d210c9e6L242-R244), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-12f648feaf3caa0382edb3b014fdd7dbedcad48bee322535d7cd01c5d210c9e6L277-R276), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-78d917ce86078797614d11e0c46bdd7f0f15ff00b3ebc25bca413370f07be21aL132-R127), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-78d917ce86078797614d11e0c46bdd7f0f15ff00b3ebc25bca413370f07be21aL156-R151), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-844617f9965e63398da7daa55eb25905d69615fd3e95c4953b8f486661b93d0aL92-R87), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-334e4791bcfe5714cefed473096aba70368d2cea07143640f8396e570b5e61f7L36-R32), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-bb71d3e7437a45c3978afff7a00abefa4e0c15a401677f1a732c773f7027d36eL120-R116), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-a0e54c8709f9bb05620ed26294aecfdbd07627aa7abb084e87ccc5a580530c71L110-R109), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-9ac0f4a6f5f6e7f8ca9a08ba42c07109b6705731c89c6b1af5daf7e2e9d5850dL92-R88), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-fb37d7abd9dddd9f2a8c076bfd9eea2336873f1051d1f37dceaca099206a4aedL351-L356), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-fb37d7abd9dddd9f2a8c076bfd9eea2336873f1051d1f37dceaca099206a4aedL421-R426), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-fb37d7abd9dddd9f2a8c076bfd9eea2336873f1051d1f37dceaca099206a4aedL435-R440), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-fb37d7abd9dddd9f2a8c076bfd9eea2336873f1051d1f37dceaca099206a4aedL446-R452), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-30afbf329ccd8dcb6bf69a0c2b4464e6ef851c61ed0d4c133113aa6afd4b8f28L68-R68), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-ef29a310ce5cad95573716c19d940bb06da30b1a8e571b8d7f6154dd8c42d92eL143-R143), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-51d36d0662438e162135ad6713e967a78491b32e953192265b2ecc71c9800e26L43-R39), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-f1df5cf9560d26c9862a4824e7d1a13f6a8bec1ec7338e9327f3e7a6a74c1585L147-R151), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-f1df5cf9560d26c9862a4824e7d1a13f6a8bec1ec7338e9327f3e7a6a74c1585L165-R164), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-f1df5cf9560d26c9862a4824e7d1a13f6a8bec1ec7338e9327f3e7a6a74c1585L177-R176), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-f1df5cf9560d26c9862a4824e7d1a13f6a8bec1ec7338e9327f3e7a6a74c1585L206-R212), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-377ed68f4e882f6438c31331026bdb11548e6b62be65727a5793cda37badfa93L110-R110), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-0257ee7455cf658a30e3d6ef9a4e98d4aeb93d6ec5d9d1421e0305718eac4dbfL127-R130), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-0257ee7455cf658a30e3d6ef9a4e98d4aeb93d6ec5d9d1421e0305718eac4dbfL141-R140))
*  Rename `RecommendsOld` component to `Banner` and change the file name and path accordingly to use a more concise and descriptive name for the component that displays the banner section of the landing page ([link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-6a484b82d5234dc2b8c7c367190b951e79e2d45ebe10433597c11365aee1c877L3-R10), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-844617f9965e63398da7daa55eb25905d69615fd3e95c4953b8f486661b93d0aL3-R7))
*  Add `NewToken` interface to declare a custom token type that extends the default token type from `antd-style` and defines some additional properties for the site theme, and use it to augment the `CustomToken` interface from `antd-style` and specify the generic type parameter for the `ThemeProvider` component ([link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-3bb66880942ca129db42a1a6c20126916fc7b9be5daf9723506df03fa96f0995R7-R25), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-3bb66880942ca129db42a1a6c20126916fc7b9be5daf9723506df03fa96f0995L19-R38))
*  Remove some properties from the `Menu` and `Card` component themes that override the default theme values and could cause inconsistency or unexpected appearance of the components ([link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-fb37d7abd9dddd9f2a8c076bfd9eea2336873f1051d1f37dceaca099206a4aedL362-L369), [link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-fb37d7abd9dddd9f2a8c076bfd9eea2336873f1051d1f37dceaca099206a4aedL395-R407))
*  Rename `colorActiveBarWidth` property to `activeBarBorderWidth` in the `Menu` component theme to match the correct property name for the component theme ([link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-fb37d7abd9dddd9f2a8c076bfd9eea2336873f1051d1f37dceaca099206a4aedL382-R377))
*  Replace `overlay` prop with `menu` prop in the `Breadcrumb.Item` component to use a more concise and consistent way to pass a menu object to the component ([link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-fb37d7abd9dddd9f2a8c076bfd9eea2336873f1051d1f37dceaca099206a4aedL471-R472))
*  Rename `useCarouselStyle` function to `getCarouselStyle` function to avoid using the `use` prefix for a function that is not a custom hook ([link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-5137c94c78aea4423103cc7e7309ec68b1d233d0dae3b03750694ea29c3206cfL102-R102))
*  Change `size` prop from `'large'` to `"large"` in the `Input` component to use double quotes instead of single quotes for string literals, which is the preferred style for TypeScript code ([link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-0257ee7455cf658a30e3d6ef9a4e98d4aeb93d6ec5d9d1421e0305718eac4dbfL141-R140))
*  Delete file `.dumi/hooks/useSiteToken.ts` as it is no longer needed ([link](https://github.com/ant-design/ant-design/pull/43623/files?diff=unified&w=0#diff-307d187d1bbfcfb5320993a59609049c6495fd91ce8f2357cbbaba952af53e82))

### CODE REVIEW

**Comment by afc163** [comment]
antd-style 之前有 SSR 过程的中的样式丢失问题，还特别改回 emotion，要注意一下：https://github.com/ant-design/ant-design/pull/43158

**Comment by MadCcc** [comment]
> antd-style 之前有 SSR 过程的中的样式丢失问题，还特别改回 emotion，要注意一下：#43158

可能是导出方式问题

File: .dumi/theme/slots/Header/SwitchBtn.tsx [hunk 1]
```diff
@@ -86,21 +84,27 @@ const useStyle = () => {
...
-};
+});
 
 const LangBtn: React.FC<LangBtnProps> = (props) => {
   const { label1, label2, tooltip1, tooltip2, value, pure, onClick } = props;
 
-  const { btn, innerDiv, labelStyle, label1Style, label2Style } = useStyle();
+  const {
+    styles: { btn, innerDiv, labelStyle, label1Style, label2Style },
+  } = useStyle();
```
Comment (arvinxx) [inline .dumi/theme/slots/Header/SwitchBtn.tsx:94]: const { styles, cx } = useStyle()


从上面可以取出 cx ，等价于 classNames

Comment (MadCcc) [reply .dumi/theme/slots/Header/SwitchBtn.tsx:94]: 习惯用 classNames 了🐶

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**